starter-code/lib/harryPotter.js

```diff
@@ -8,3 +8,32 @@
                    "4 October", "Filius Flitwick", "17 October", "Molly Weasley",
                    "30 October", "Bill Weasley", "29 November", "Rubeus Hagrid",
                    "6 December", "Charlie Weasley", "12 December"];
+
+                   var moreBirthdays = ["Lily Evans", "30 January", "James Potter", "27 March",
+                                        "Dudley Dursley", "30 June", "Tom Riddle", "31 December"];
+
+var pairs = [];
+
+function characterBirthdays(array) {
+  for(var i = 0; i < array.length; i += 2){
+    pairs.push(_.slice(array, i, i+2));
+  }
+  return pairs;
+};
+
+function addBirthdays(array) {
+  var chunks = _.chunk(array, 2);
+  chunks.forEach(function(item){
+    pairs.push(item);
+  });
+}
+
```

Comment:
You could use the chunk and concat methods:

var pairsOfBirthdays = _.chunk(birthdays, 2);
var allBirthdays = function () { return _.concat(pairsOfBirthdays, _.chunk(moreBirthdays, 2));};